posthog/api/__init__.py

```diff
@@ -512,11 +511,11 @@ def register_grandfathered_environment_nested_viewset(
     ["team_id"],
 )
 
-register_grandfathered_environment_nested_viewset(
-    r"alerts",
-    alert.AlertViewSet,
-    "environment_alerts",
-    ["team_id"],
-)
+# register_grandfathered_environment_nested_viewset(
```

Comment:
It is expected?

Reply:
oh yes thanks for flagging, wanted to check with @Twixes if we still want this?